go/noise/gaussian_noise.go

```diff
@@ -155,6 +181,23 @@ func addGaussian(x, sigma float64) float64 {
 	return roundToMultipleOfPowerOfTwo(x, granularity) + float64(sample)*granularity
 }
 
+// computeConfidenceIntervalGaussian computes a confidence interval that contains the raw value x from which
+// float64 noisedX is computed with a probability equal to 1 - alpha with the given sigma
+func computeConfidenceIntervalGaussian(noisedX, sigma, alpha float64) ConfidenceInterval {
+	z := inverseCDFGaussian(sigma, alpha/2)
```

Comment:
why z? Can the name be more meaningful?

Reply:
At first i chose shiftingValue for the name, but Dennis suggested something shorter and consistent with the Laplace implementation. Neither me, nor him, had any better ideas for the name at the moment.